index.d.ts

```diff
@@ -58,7 +55,49 @@ declare const tempy: {
 	directory(): string;
 
 	/**
-	Get the root temporary directory path. For example: `/private/var/folders/3x/jf5977fn79jbglr7rk0tq4d00000gn/T`.
+	Deletes temporary directories and returns an array of deleted path.
+
+	@returns An array of deleted paths.
+
+	@example
+	```
+	import tempy = require('tempy');
+
+	tempy.clean();
+	//=> ['/private/var/folders/3x/jf5977fn79jbglr7rk0tq4d00000gn/T/4f504b9edb5ba0e89451617bf9f971dd', ...]
+	```
+	*/
+	clean(): string[];
+
+	/**
+	Returns a `Promise` for value obtained in `task`.
+
+	@param task - A function that will be called with a temporary directory path. The directory is created and deleted when `Promise` is resolved.
+	@returns Task output
+
+	@example
+	```
+	import pathExists = require('path-exists');
+	import tempy = require('tempy');
+
+	(async () => {
+		console.log(await tempy.job(directory => pathExists(directory)));
```

Comment:
This is not actually showing off how useful `.job` is. The example does absolutely nothing.